src/app/services/query.service.ts

```diff
@@ -180,7 +192,23 @@ export class QueryService {
       );
   }
 
-  /**
+  private mergeInclusionAndExclusionCounts(initialUpdate?: boolean) {
+    if (this.autosaveSubjectSets) {
+      // Not computing the counts based on inclusion and exclusion counts,
+      // but using the subject set and counts per study concept response instead.
+      return;
+    }
+    this.subjectCount_1 = this.inclusionSubjectCount - this.exclusionSubjectCount;
+    this.observationCount_1 = this.inclusionObservationCount - this.exclusionObservationCount;
```

Comment:
so if inclusionObservationCount < exclusionObservationCount, we will have observationCount < 0? Or it is not possible?

Reply:
It is not possible by construction: the exclusion constraint is of the form (inclusionConstraint AND NOT (...)), so will always be maximum the number of elements of the inclusion result.